src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,172 @@
+CockroachDB is made available under the [CockroachDB Software License][csl].
+
+## Types of licenses
+
+The table of licenses below refers to options for {{ site.data.products.core }} deployments. All {{ site.data.products.cloud }} deployments automatically have a valid **Enterprise** license.
```

Comment:
@rmloveland what does the second line here mean?

Reply:
Basically if you use our cloud product your cloud-deployed binary already has an Enterprise license via your cloud situation so you don't have to worry / think about this

However it might not be necessary or even really make sense. Since it caught your eye I'm inclined to remove it because it may be saying "too much" about something that is not really necessary.

What do you think of the idea of removing it?